docs/library/pyb.Pin.rst

```diff
@@ -62,12 +62,6 @@ When a pin has the ``Pin.PULL_UP`` or ``Pin.PULL_DOWN`` pull-mode enabled,
 that pin has an effective 40k Ohm resistor pulling it to 3V3 or GND
 respectively (except pin Y5 which has 11k Ohm resistors).
```

Comment:
This paragraph can also go (or be moved), it doesn't belong here.  Maybe just note that pull up/down are 40k Ohm in the `init()` method below where it defines the valid values of the `pull` argument.

Reply:
Paragraphed moved to `init()` section.